### PR TITLE
Ezrinsky pbi 189417 doc qa edgecmd 1.2

### PR DESCRIPTION
**Pull request by osisoft-ezrinsky**
Completed doc QA for two topics:

* Configure an adapter or EDS with commands
* EdgeCmd commands

Overall, no major changes, just a few tweaks for commas and articles to make complete sentences.

### CODE REVIEW

File: V1.2/Configuration/configure-an-adapter-or-eds-with-commands.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Complete the following steps to change all values of a facet:
 1. Access EdgeCmd utility through the command line.
 2. Type the `set` keyword.
 3. Add the `<facetName>`and `<componentId>`, replacing both with their respective values.
-4. Add all key=value pairs. Then press Enter.
+4. Add all key=value pairs. Then, press Enter.
```
Comment (osisoft-ezrinsky) [inline V1.2/Configuration/configure-an-adapter-or-eds-with-commands.md:29]: I wasn't certain if we should offset "key=value" with code styling to make it stand out a little.

File: V1.2/Configuration/configure-an-adapter-or-eds-with-commands.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ Complete the following steps to change all values of a facet:
 1. Access EdgeCmd utility through the command line.
 2. Type the `set` keyword.
 3. Add the `<facetName>`and `<componentId>`, replacing both with their respective values.
```
Comment (osisoft-ezrinsky) [inline V1.2/Configuration/configure-an-adapter-or-eds-with-commands.md:28]: We describe `set` as a keyword. Do we need a name for `factName` or `componentId` ?

File: V1.2/Configuration/configure-an-adapter-or-eds-with-commands.md (hunk 1 above)
Comment (TinaUll) [reply V1.2/Configuration/configure-an-adapter-or-eds-with-commands.md:29]: I think code styling is not necessary here, it's really just a term that is not actually used in the code

**Comment by TinaUll** [comment]
No questions, thanks @ericzrinsky.

**Review by TinaUll — APPROVED** [review]

**Pull request merged**